sdk/tvl-usd.js

```diff
@@ -0,0 +1,20 @@
+const debug = require("debug")("opentvl:tvl-usd");
+
+const TOKEN_PRICES = require('./data/tokenPrices.json');
+
+function computeTVLUSD(tokenCounts) {
```

Comment:
This file as well as the token prices json maybe belong somewhere outside of the sdk directory?

Reply:
With chainlink we won't have tokenPrices.json anymore.